docs/dependencies/images.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Publishing to the images namespace"
+sidebar_label: image packages
+---
+
+Eik's provides an image namespace for uploading images. 
+The purpose of this namespace is three-fold. 
+
+1. By creating a separate namespace, files that change infrequently (images and other media) can be uploaded and versioned separately from files that change often (JavaScript and CSS)
+2. An image manipulation service such as Fastly's [image optimization](https://www.fastly.com/products/image-optimization) can be placed in front of this namespace.
+3. Your Eik server can be configured to allow larger package sizes on the image namespace and smaller package sizes on other namespaces.
+
+In this document you'll learn how to:
+
+- publish images to your Eik server
+- how to update existing images
+- create an alias that can be updated as you publish new patch or minor versions of your images
+
```

Comment:
Would it make sense to have an addition point where we show how this new image namespace works in conjunction with regular ESM publishing. I think it's likely that people would not have dedicated repos for images, so showing how it would work in a real life project I think would be useful.